[2022-08-19 23:11:26,315] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T07:10:00+00:00 [queued]>
[2022-08-19 23:11:26,325] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T07:10:00+00:00 [queued]>
[2022-08-19 23:11:26,326] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:11:26,327] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 23:11:26,328] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:11:26,339] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_telegram_version> on 2020-01-01 07:10:00+00:00
[2022-08-19 23:11:26,349] {standard_task_runner.py:52} INFO - Started process 4568 to run task
[2022-08-19 23:11:26,365] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sefa_dag', 'get_telegram_version', 'scheduled__2020-01-01T07:10:00+00:00', '--job-id', '881', '--raw', '--subdir', 'DAGS_FOLDER/sefa_dag.py', '--cfg-path', '/tmp/tmpeoyap9vp', '--error-file', '/tmp/tmp8ja_yhxg']
[2022-08-19 23:11:26,368] {standard_task_runner.py:80} INFO - Job 881: Subtask get_telegram_version
[2022-08-19 23:11:26,474] {task_command.py:371} INFO - Running <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T07:10:00+00:00 [running]> on host fd7a784b9251
[2022-08-19 23:11:26,562] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sefa_dag
AIRFLOW_CTX_TASK_ID=get_telegram_version
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T07:10:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T07:10:00+00:00
[2022-08-19 23:11:26,564] {python.py:173} INFO - Done. Returned value was: 1.24.0
[2022-08-19 23:11:26,587] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=sefa_dag, task_id=get_telegram_version, execution_date=20200101T071000, start_date=20220819T231126, end_date=20220819T231126
[2022-08-19 23:11:26,621] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 23:11:26,645] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
